test/.eslintrc.js

```diff
@@ -0,0 +1,37 @@
+const path = require('path');
+
+module.exports = {
+    env: {
+        browser: true,
+        es6: true,
+    },
+    extends: [
+        "eslint:recommended",
+        "plugin:@typescript-eslint/eslint-recommended",
+        "plugin:@typescript-eslint/recommended",
+        "plugin:@typescript-eslint/recommended-requiring-type-checking",
+        "airbnb-base",
+        "../scripts/config/.eslintrc.js"
+    ],
+    parser: "@typescript-eslint/parser",
+    parserOptions: {
+        ecmaVersion: 2018,
+        sourceType: "module",
+        tsconfigRootDir: __dirname,
+        project: [
+            path.resolve(__dirname, "../tsconfig.json"),
+          ]
+        },
+    rules: {
```

Comment:
Can we put these rules into the devtools repo and have this (and the corresponding file in the View repo) extend it?